include/grpc/event_engine/event_engine.h

```diff
@@ -520,6 +520,46 @@ void AbslStringify(Sink& out, const EventEngine::TaskHandle& handle) {
   out.Append(detail::FormatHandleString(handle.keys[0], handle.keys[1]));
 }
 
+/// Set the default EventEngine instance, which will be used throughout gRPC
+///
+/// gRPC will hold a ref to this engine. For your engine to be shut down, you
+/// must call \a ShutdownDefaultEventEngine at the end of your program.
+///
+/// Earlier calls to \a GetDefaultEventEngine will still hold a ref to the
+/// previous default engine instance, if any.
+void SetDefaultEventEngine(std::shared_ptr<EventEngine> engine);
+
+/// Returns the shared default EventEngine instance.
+///
+/// GetDefaultEventEngine is a lazy thing: either a shared global EventEngine
+/// instance exists and will be returned, or that shared global instance will be
+/// created and returned. The returned shared_ptr<EventEngine>'s life is
+/// determined by the shared_ptr, and therefore EventEngines may be created and
+/// destroyed multiple times through the life of your gRPC process, there is no
+/// guarantee of one persistent global instance like in iomgr.
```

Comment:
We shouldn't be referring to iomgr in this API documentation, since (a) it's not a public API and (b) it's going away.